### DISCUSSION TITLE
`nx release plan` against packages without changes should default to 'none'

### DISCUSSION THREAD

**Discussion by jonathanmorley** [nrwl/nx#27008] [Feature Requests]
The default option (because it is the first) when running `nx release plan` is `major`.

Often, changes only affect a subset of packages. It is useful to present that set as the default set of packages to make material (not none) bumps.

**Comment by bitfrost** [comment]
Also, ideally nx release plan should only default to prompting for the same packages returned as needing plans form nx release plan:check ,  walking through the rest is painful in a large mono repo
